app/assets/stylesheets/sass/users_styles/show.scss

```diff
@@ -11,7 +11,7 @@
   }
 }
 
-.student__container--outer {
+.main-content__container--inner {
```

Comment:
Selector `main-content__container--inner` should be written in lowercase with hyphens